src/signals/incident/definitions/wizard-step-3-telefoon.js

```diff
@@ -12,11 +13,15 @@ export default {
           subtitle: 'Zo kunt u ons helpen het probleem sneller of beter op te lossen.',
           path: 'reporter.phone',
           placeholder: 'Telefoonnummer',
-          type: 'text'
+          type: 'text',
+          autoRemove: /[^\d+-]/g
         },
         render: FormComponents.TextInput,
         options: {
-          validators: [Validators.maxLength(17)]
+          validators: [
+            Validators.maxLength(17),
+            validatePhoneNumber
```

Comment:
Now that illegal phone number characters are automatically removed, there is not much use for this validator